fairseq_cli/preprocess.py

```diff
@@ -331,7 +331,7 @@ def make_all_alignments():
         ) as f:
             for k, v in align_dict.items():
                 print("{} {}".format(src_dict[k], tgt_dict[v]), file=f)
-
+    logging.shutdown()
```

Comment:
According to python docs: "This should be called at application exit and no further use of the logging system should be made after this call." However in a unittest context (or if there's library usage) we may still call into logging after this.

I assume the motivation is to close the FileHandler registered above? Can we instead save a reference to the FileHandler and do:
```
file_handler.close()
logger.removeHandler(file_handler)
```

Reply:
Likely yes - I can update.